apps/foundation-demo/src/app/checkout/components/shipping-form/shipping-form.component.html

```diff
@@ -0,0 +1,49 @@
+<form class="shipping-form" [formGroup]="form" #ngForm="ngForm" (ngSubmit)="onSubmit(form)">
+  <div class="shipping-form__title">
+    shipping details
+  </div>
+  <input type="text" class="shipping-form__first-name"
+    placeholder="First Name"
+    name="firstname"
+    [formControl]="firstname"
+    [class.shipping-form__error]="firstname.errors && (firstname.touched || ngForm.submitted)"
```

Comment:
We should consider migrating form styles into their own separate module.

Reply:
done